src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -343,6 +349,16 @@ const SingleActiveFIMap = (props: MapSingleFIProps & RouteComponentProps<RoutePa
 
   const mapBounds = getMapBounds(jurisdiction);
 
+  function getLineColor(decimalValue: number) {
```

Comment:
Do you think that we should make [something like this](https://github.com/onaio/reveal-frontend/blob/8fcee000dc30c07ece47f898b9962b226cc92b0a/src/configs/settings.ts#L1617) for FI and then use it here?

Reply:
Yeap, we can and pass `lineColorThreshold` instead of `lineColor`

Reply:
Cool, lets make one for Focus Investigations.  You can make it in the same settings file and can copy how we did it for [Namibia IRS](https://github.com/onaio/reveal-frontend/blob/8fcee000dc30c07ece47f898b9962b226cc92b0a/src/configs/settings.ts#L1644) if it makes sense.